source/agora/consensus/protocol/Nominator.d

```diff
@@ -624,11 +628,6 @@ extern(D):
 
     private void handleSCPEnvelope (in SCPEnvelope envelope) @trusted
     {
-        // ignore messages if `startNominatingTimer` was never called or
-        // if `stopNominatingTimer` was called
-        if (this.nomination_timer is null)
-            return;
-
```

Comment:
Shouldn't this removal be in another commit or the second commit because it changes the handling algorithm?

Reply:
Didn't work as expected so is reverted.

Reply:
Strangely `ValidatorCleanRestart` passes without removing this :thinking: